frontend/src/metabase/components/MetadataInfo/TableInfoPopover/TableInfoPopover.tsx

```diff
@@ -16,20 +15,12 @@ interface TableSubset {
   description?: string;
 }
 
-const propTypes = {
-  table: PropTypes.shape({
-    id: PropTypes.oneOf([PropTypes.number, PropTypes.string]),
```

Comment:
I think propTypes are useful for this component because it ended up getting used in a bunch of `.jsx` files (I think it's only used in one place right now, but was and maybe will be used in more files at some point).

There's an error here that'll be noisy -- this should be `PropTypes.oneOfType` instead of `PropTypes.oneOf`. Not sure if there are other issues with this.